### ISSUE TITLE
Issue with paths in webjar scss

### ISSUE THREAD

**Issue by cvrabie** [ShaggyYeti/sbt-sass#15]
I am able to bundle scss from a webjar into my main css, but there are problems bundling scss from webjars that reference other resources from that webjar. For example font-awesome  will get downloaded to `lib/font-awesome` with the fonts in locations like `/assets/lib/font-awesome/fonts/fontawesome-webfont.eot`. The problem is that the font-awesome scss references it with `../fonts/fontawesome-webfont.eot` which after compilation will be `/assets/fonts/fontawesome-webfont.eot`.

Now I'm not sure that this issue should be fixed by the sbt-sass plugin, by configurations to sass itself or by some other means. I've opened a [stack overflow question](http://stackoverflow.com/questions/26495287/sass-includes-and-webjars) trying to get some opinions, but so far no luck. However, It's my opinion that this can/should be solved at this level. At the very list you can add a note to the _troubles and solutions_ section in the read me file, as I expect this will become a more widespread issue together with the prevalence of webjars.

**Comment by tindzk** [comment]
I don't think this is an sbt-sass issue. Usually, `.scss` files provide a way to override paths. As for Bootstrap, I had to use the following:

``` scss
$icon-font-path: "lib/bootstrap-sass/fonts/bootstrap/"
@import "lib/bootstrap-sass/stylesheets/bootstrap.scss"
```
